pkg/common/catalog/catalog_test.go

```diff
@@ -319,13 +319,22 @@ func (s *CatalogSuite) TestPluginsFill() {
 				r.NoError(ps.Fill(c))
 			},
 		},
+		{
+			"ignore struct tag",
```

Comment:
It would be great if we can add a test cases to exercise the `not expecting key=value` (for "-") and `expected key=value` (for "min" and "max") errors.